content/are-there-any-paid-versions-of-tor-browser/contents.lr

```diff
@@ -0,0 +1,15 @@
+_model: question
+---
+title: Are there any paid versions of Tor Browser?
+---
+seo_slug: are-there-any-paid-versions-of-tor-browser
+---
+description:
+
+No, Tor Browser is an open source software and it is free.
+Any browser forcing you to pay or donate and is claiming to be Tor browser is fake.

```

Comment:
Please write it Tor Browser, capital B.

Reply:
Please suggest name, I used the existing conventions in project and tried to make it self explanatory. If there is no technical issues of a bit longer name then I think we keep it this way :)